src/Microdown/MicAbstractAnnotatedBlock.class.st

```diff
@@ -18,14 +18,18 @@ MicAbstractAnnotatedBlock >> addLineAndReturnNextNode: line [
 		ifTrue: [ ^ self ].
 	label 
 		ifNil: [ 
-			| indexOfFirstSpace |
-			indexOfFirstSpace := line indexOf: Character space.
-			label := indexOfFirstSpace = 0
-				ifTrue: [ '' ]
-				ifFalse: [line copyFrom: self lineMarkup size + 1 to: indexOfFirstSpace - 1.].
-			body := (line copyFrom: indexOfFirstSpace + 1 to: line size) trim.]
-		ifNotNil: [ body = '' ifTrue: [ body := body , line ]
-			ifFalse: [ body := body , String cr , line ]]
+			| indexOfFirstClosingBracket |
+			indexOfFirstClosingBracket := line indexOf: $].
```

Comment:
I wonder what is the impact if the line does not contain ]

Reply:
After getting the index of the first ], I test if he egal to 0.
If true, I removed the AnnotatedParagraphe to MicRootBlock, and I replace by a MicBlockQuoteBlock

indexOfFirstClosingBracket = 0
				ifTrue: [ parent children removeLast. ^ self createMicBlockQuoteBlock: line ]